hypertrace-graphql-labels-schema/src/main/java/org/hypertrace/graphql/label/request/LabelCreateRequest.java

```diff
@@ -0,0 +1,10 @@
+package org.hypertrace.graphql.label.request;
+
+import org.hypertrace.core.graphql.context.GraphQlRequestContext;
+import org.hypertrace.graphql.label.schema.mutation.CreateLabel;
+
+public interface LabelCreateRequest {
```

Comment:
nit: on all new request objects, we should implement ContextualRequest and let that bring in the context.